debian/marquee/03-nginx-conf

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+apt install nginx
+
+# a2dissite doesn't work if nothing is enabled
+rm -f /etc/nginx/sites-enabled/*
+
+cp -p nginx/conf/http.conf /etc/nginx/conf.d/
+cp -p nginx/conf/whatwg.conf /etc/nginx/
+cp -p nginx/conf/whatwg-headers.conf /etc/nginx/
+
+chown -R deploy:deploy /var/www
+rm -rf /var/www/html
+
+/usr/sbin/nginx -t
+systemctl start nginx.service
```

Comment:
It will be running after the install, so restart. And is this different from `nginx -s reload`?

Reply:
> It will be running after the install, so restart.

OK

> And is this different from `nginx -s reload`?

Yes, `reload` causes it to just re-read/re-evaluate the config, without restarting. But if nginx is for some reason not running at the point, then `reload` won’t start it but I guess will just fail.

But even if nginx is stopped `restart` will cause it start (after loading the config).

So it seems like `restart` is what we do want here, right?